src/components/buttons/buttons.html

```diff
@@ -416,9 +350,8 @@ <h3 class="docs-block__header">Secondary button wide</h3>
     <div class="docs-block__content">
         <button class="sg-button-secondary sg-button-secondary--full">This button is wide</button>
 
-        <button
-            class="sg-button-secondary sg-button-secondary--alt-inverse sg-button-secondary--small sg-button-secondary--full">
+        <button class="sg-button-secondary sg-button-secondary--alt-inverse sg-button-secondary--small sg-button-secondary--full">
             This one to!
         </button>
     </div>
-</section>
+</section>
```

Comment:
please add an empty line a the end of the file ( I think we have this defined in editorconfig )